dataactcore/scripts/initialize.py

```diff
@@ -120,6 +123,7 @@ def main():
     parser.add_argument('-c', '--load_agencies', help='Update agency data (CGACs, FRECs, SubTierAgencies)',
                         action='store_true')
     parser.add_argument('-t', '--update_tas', help='Update broker TAS list', action='store_true')
+    parser.add_argument('--cfda_load', help='Pull Catalog of Federal Domestic Assistance', action='store_true')
```

Comment:
where is the short version of the argument? Everything else has one. Perhaps `-cfda` is a valid flag since we already have a `-c`